subprojects/osgi-test/src/testBundle/java/org/mockito/osgitest/testbundle/MockNonPublicClassFailsTest.java

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright (c) 2019 Mockito contributors
+ * This program is made available under the terms of the MIT License.
+ */
+package org.mockito.osgitest.testbundle;
+
+import org.mockito.exceptions.base.MockitoException;
+
+import static org.mockito.Mockito.mock;
+
+public class MockNonPublicClassFailsTest implements Runnable {
+
+    static class NonPublicClass {}
+
+    @Override
+    public void run() {
+        try {
+            NonPublicClass nonPublicClass = mock(NonPublicClass.class);
+            throw new AssertionError();
+        } catch(MockitoException e) {
```

Comment:
Nit: don't use `try-catch` clauses in tests. Instead, use `assertThat` or `assertThrows`

Reply:
Using `@Test(expected = MockitoException.class)` now that it's available.